src/panoptoindexconnector/connector.py

```diff
@@ -334,6 +335,8 @@ def sync(config, last_update_time):
     except requests.exceptions.HTTPError as ex:
         LOG.exception('Received error response %s | %s', ex.response.status_code, ex.response.text)
         exception = ex
+    except CustomExceptions.QuotaLimitExceededError as ex:
```

Comment:
Don't we need to log here, as we do for other error cases?

Reply:
No need to log since it will be logged in caller method. I have added a comment for that. Thanks.